.github/workflows/release.yml

```
@@ -0,0 +1,62 @@
+            - uses: actions/setup-node@v7
+              with:
+                  node-version-file: '.node-version'
+                  cache: 'npm'
+                  registry-url: 'https://registry.npmjs.org'
+
+            - name: Install dependencies
+              run: npm ci --ignore-scripts
+
+            - name: Check for releasable commits
+              id: gate
+              run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:6:14: Double quote to prevent globbing and word splitting

(shellcheck)